### PR TITLE
Adding an issue template to github

### PR DESCRIPTION
**Pull request by shrit**
Signed-off-by: Omar Shrit <omar@shrit.me>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Use this template for reporting a bug or to help us improve ei-install-scripts.
+title: ''
+labels: ''
+assignees: ''
+
```
[the remark this answers is not in the document]
Comment (shrit) [reply .github/ISSUE_TEMPLATE/bug_report.md:7]: This is a YAML front matter it is required for github to recognize the template, see here https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/manually-creating-a-single-issue-template-for-your-repository

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+---
+
+### Issue description
+
```
[the remark this answers is not in the document]
Comment (shrit) [reply .github/ISSUE_TEMPLATE/bug_report.md:11]: I will remove it if you think that the expected and current behavior will be enough

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ijdoc) [reply .github/ISSUE_TEMPLATE/bug_report.md:7]: oh! it lets us pre-fill content and help select amongst multiple templates... very cool!

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ijdoc) [reply .github/ISSUE_TEMPLATE/bug_report.md:11]: I think so, we can add it later if we find the rest to be insufficient

**Comment by ijdoc** [comment]
oh man, the front matter forces an extra click and makes it possible to start an empty issue, so I will try it without the front matter and see what happens

**Pull request merged**